pkg/fftypes/node_status.go

```diff
@@ -43,3 +44,18 @@ type NodeStatusOrg struct {
 type NodeStatusDefaults struct {
 	Namespace string `ffstruct:"NodeStatusDefaults" json:"namespace"`
 }
+
+// NodeStatusPlugins is a map of plugins configured on the node
+type NodeStatusPlugins struct {
+	Blockchain    []*NodeStatusPlugin `ffstruct:"NodeStatusPlugins" json:"blockchain"`
+	Database      []*NodeStatusPlugin `ffstruct:"NodeStatusPlugins" json:"database"`
+	DataExchange  []*NodeStatusPlugin `ffstruct:"NodeStatusPlugins" json:"dataExchange"`
+	Identity      []*NodeStatusPlugin `ffstruct:"NodeStatusPlugins" json:"identity"`
+	SharedStorage []*NodeStatusPlugin `ffstruct:"NodeStatusPlugins" json:"sharedStorage"`
+	Tokens        []*NodeStatusPlugin `ffstruct:"NodeStatusPlugins" json:"tokens"`
+}
+
+// NodeStatusPlugin is information about a plugin
+type NodeStatusPlugin struct {
+	Connection string `ffstruct:"NodeStatusPlugin" json:"connection"`
```

Comment:
Think the set of changes is:

- Should add `events` as a plugin category - missing at the moment
- Remove `connection` from `NodeStatusPlugin`
- Add `name` to `NodeStatusPlugin` - will only be set for plugins that accept multiple entries - currently that's just tokens (sure it will be more in the future)
- Add `pluginType` to `NodeStatusPlugin` - which will be set for all entries, and is what the `Name()` function returns from the plugin.